config/samples/resources/cloudidsendpoint/cloudids_v1beta1_cloudnetwork-dep.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: compute.cnrm.cloud.google.com/v1beta1
+kind: ComputeNetwork
+metadata:
+  name: computenetwork-dep
```

Comment:
Name should be `cloudidsendpoint-dep`. See naming convention: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/README.Samples.md#dependency-naming-convention.